controlplane/build/Dockerfile.nsmd

```diff
@@ -6,12 +6,10 @@ ENV PACKAGEPATH=github.com/networkservicemesh/networkservicemesh/
 ENV GO111MODULE=on
 
 RUN mkdir /root/networkservicemesh
-ADD ["go.mod","/root/networkservicemesh"]
-ADD ["./scripts/go-mod-download.sh","/root/networkservicemesh"]
+ADD [".","/root/networkservicemesh"]
```

Comment:
This was minimized to prevent docker stage rebuild in case of go mod not changed. So I suppose we need to return it back.

Reply:
All mod files are linked to each other
```
go: parsing ../dataplane/go.mod: open /root/networkservicemesh/dataplane/go.mod: no such file or directory
go: parsing ../sdk/go.mod: open /root/networkservicemesh/sdk/go.mod: no such file or directory
go: parsing ../go.mod: open /root/networkservicemesh/go.mod: no such file or directory
```
The only way to copy each separate mod file into docker build